src/bitcoin-cli.cpp

```diff
@@ -315,6 +316,8 @@ class GetinfoRequestHandler: public BaseRequestHandler
         result.push_back(JSONRPCRequestObj("getblockchaininfo", NullUniValue, ID_BLOCKCHAININFO));
         result.push_back(JSONRPCRequestObj("getwalletinfo", NullUniValue, ID_WALLETINFO));
         result.push_back(JSONRPCRequestObj("getbalances", NullUniValue, ID_BALANCES));
+        UniValue params{RPCConvertValues("getmempoolinfo", {"[1,10,100,200,1000]"})};
```

Comment:
Playing devil's advocate: Why to use these values and not different ones? 

I think it's very hard to argue that these values are the best possible. Without more information in PR's description, this feels arbitrary. And even if you can persuade people for Bitcoin mainnet chain, I would be surprised if these values were good for Bitcoin testnet chain.

I don't really know how to approach this but it seems important. This is also the reason why #21422 lets user to decide. Original #15836 [mentioned log scale](https://github.com/bitcoin/bitcoin/pull/15836/files#diff-decae4be02fb8a47ab4557fe74a9cb853bdfa3ec0fa1b515c0a1e5de91f4ad0bR1497) which seems in principle much better to me.

Reply:
> Playing devil's advocate: Why to use these values and not different ones?

Different can be either random values which is difficult to support in this feature which is just mean to be a summary of mempool. Or use more values in between which will use more space hence used all multiples of 10 until 100 and then trying to include everything else in 'Above 200' category.

> And even if you can persuade people for Bitcoin mainnet chain, I would be surprised if these values were good for Bitcoin testnet chain.

I tied them on testnet and mainnet. Works fine because it wont print a category if it has zero size.

> This is also the reason why #21422 lets user to decide.

This is not replacement for 21422 but a usecase for it.

> Original #15836 mentioned log scale which seems in principle much better to me.

It was closed by one of the maintainers with comment: "Closing this, given it's been taken over in #21422"


![image](https://user-images.githubusercontent.com/13405205/141219327-18086649-6c4a-43df-b208-ca849efa9f72.png)

Reply:
> > And even if you can persuade people for Bitcoin mainnet chain, I would be surprised if these values were good for Bitcoin testnet chain.
> 
> I tried them on testnet and mainnet. Works fine because it wont print a category if it has zero size.

I'm not sure we understand each other. Anyway, I just wanted to point out that any fixed intervals may sooner or later become obsolete. Suppose that your PR is merged and for whatever reasons nobody will mine fee rates less than 200. Then you will just show one bar (right?) and that's not useful really. That is my main concern and that's why I mentioned #15836 and and the log scale (a nice curve with interesting mathematical properties) - again, I'm not saying "use this", just trying to say that some math analysis might be needed here.

> > This is also the reason why #21422 lets user to decide.
> 
> This is not replacement for 21422 but a usecase for it.

Yes, I know.

> One example of fee rate distribution from https://btc.bitaps.com/:
> 
> ![image](https://user-images.githubusercontent.com/13405205/141219327-18086649-6c4a-43df-b208-ca849efa9f72.png)

I like it but it's a website and they can change the output whenever they like. I don't consider this to be true for Bitcoin Core.